test/evaluate_ecr_scan.py

```diff
@@ -0,0 +1,51 @@
+import argparse
```

Comment:
Is there a reason to write this as a standalone script as opposed to a pytest function? Does it run in parallel across images? If it needs to be a standalone script, let's call out a justification for the decision in the docstring

Reply:
This was experimental code that I uploaded to give more context, during intermediate commits. I have deleted it now.